btree/btree.c

```diff
@@ -9368,6 +9950,8 @@ static int ut_btree_kv_get_cb(struct m0_btree_cb *cb, struct m0_btree_rec *rec)
 
 			if (key != value)
 				check_failed = true;
+			// if (value == key+1)
```

Comment:
Can you specify the necessity of there comments?

Reply:
Removed these lines, I had added this for debugging purposes.